src/utils/dom.ts

```diff
@@ -100,6 +100,20 @@ export function getDescribedByElement<T extends Element>(element: Element): T |
   return (id && document.getElementById(id)) || null;
 }
 
-export function hasLabel(labelEl: HTMLCalciteLabelElement, el: HTMLElement) {
+export function hasLabel(labelEl: HTMLCalciteLabelElement, el: HTMLElement): boolean {
   return labelEl.contains(el);
 }
+
+export function addEventListenerOnce(
```

Comment:
Nitpick: Since this is a custom util maybe we could shorten the name? `listenOnce`, `once`, `onOnce`?